docs/docs_server_configuration_reference.md

```diff
@@ -144,6 +144,24 @@ Searches for `index.js` if a directory specified.
 }
 ```
 
+## startPoints
+
+Contains list of spawn points.
```

Comment:
If I understand everything correctly, actual spawn point is chosen randomly every time. Let's add info about that, because just 'list of spawn points' is not very clear